src/sugar/cli.py

```diff
@@ -1,104 +1,26 @@
 """Definition of the CLI structure."""
-import argparse
+from __future__ import annotations
+
 import os
 import sys
 
 from pathlib import Path
-from typing import Tuple
+from typing import Any, List, Tuple
```

Comment:
at the beginning you have `from __future__ import annotations`

so here you don't need to import `List` anymore.
in you code you can annotate the type with `list`, for example: `list[str]`

`List` is deprecated

Reply:
Oh sure, I'll change that